documentation/modules/upgrading/proc-upgrade-brokers-newer-kafka.adoc

```diff
@@ -67,6 +68,9 @@ spec:
 ----
 <1> This is changed to the new version
 <2> This remains at the current version
+--
++
+WARNING: You cannot downgrade Kafka once `spec.kafka.version` has changed, as the `inter.broker.protocol.version` for the new Kafka version will differ. The protocol version determines the schemas used for persistent metadata stored by the broker, including messages written to `__consumer_offsets`. The cluster will not understand the messages.
```

Comment:
```suggestion
WARNING: You cannot downgrade Kafka after `spec.kafka.version` has changed, because the `inter.broker.protocol.version` for the new Kafka version will differ. The inter-broker protocol version determines the schemas used for persistent metadata stored by the broker, including messages written to `__consumer_offsets`. The downgraded cluster will not understand the messages.
```